src/main/java/ru/ayaz/ChatServer.java

```diff
@@ -50,7 +50,7 @@ private void sendMessageToEveryone(String message, String sender) {
             String nickname = entry.getKey();
             User user = entry.getValue();
             if (!nickname.equals(sender)) {
-                if (!user.isInIgnoredSet(sender)) {
+                if (!user.isIgnored(sender)) {
```

Comment:
&#x1F34E; **Issue**
Better combine 2 if statements into a single if statement and take out the combined condition into a separate function with a meaningful name.